src/components/NewProductsSection/NewProductsSection.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { SliderMedium } from "../SliderMeduim/SliderMedium";
+import { ContainerSC, TitleSC, WrapperSC, SliderSC } from "./style";
+import { useTranslation } from "react-i18next";
+import Product1Image from "/public/icons/product1.svg";
+import Product2Image from "/public/icons/product2.svg";
+import Product3Image from "/public/icons/product3.svg";
+import Product4Image from "/public/icons/product4.svg";
+import Product5Image from "/public/icons/product5.svg";
+import Product6Image from "/public/icons/product6.svg";
+
+const data = [
```

Comment:
separate file with constants

Reply:
we cannot move it cuz there are tsx component